docker_build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Clone expressapi repository to the nameless directory
+if [ -d /home/jasondoze/nameless/package.json ]
+then
+  echo -e "\n==== Expressapi app present ====\n"
+else
+  echo -e "\n==== Cloning expressapi repository to nameless directory ====\n"
+  git clone https://github.com/Jason-Doze/expressapi.git 
+fi
+
```

Comment:
How would assertions look on all the code below? Why didn't you try checking before doing any of this?